os_builders/roles/nubes_bootcontext/tasks/main.yml

```diff
@@ -0,0 +1,36 @@
+- name: Copy in nubes-bootcontext script
+  copy:
+    src: "nubes-bootcontext.sh"
+    dest: "/usr/local/sbin/nubes-bootcontext.sh"
+    owner: root
+    group: root
+    mode: 0755
+
+- name: Copy in nubes-boot systemd unit
+  copy:
+    src: "nubes-boot.service"
+    dest: "/etc/systemd/system/nubes-boot.service"
+    owner: root
+    group: root
+    mode: 0755
+
+
+- name: Copy in motd
+  copy:
+    src: "motd"
+    dest: "/etc/motd"
+    owner: root
+    group: root
+    mode: 0644
+
+- name: Start systemd service
+  become: true
+  ansible.builtin.command:
+    cmd: "systemctl enable nubes-boot.service"
```

Comment:
Nit: Could we move these steps into a dedicated `nubes_prep.yml` (or similar)
Then this file can simply be:

```
- include_tasks: nubes_prep.yml
- include_tasks: update_cloud_users.yml
```

Reply:
I believe I do in the next PR